rubin_sim/maf/run_comparison/gather_summaries.py

```diff
@@ -47,33 +123,19 @@ def gather_summaries():
     )
     args = parser.parse_args()
 
-    # Identify a subset of MAF directories if suffix is set
-    if args.suffix is None:
-        run_dirs = None
-    else:
-        run_dirs = glob.glob(f"*{args.suffix}")
+    run_dirs = glob.glob(args.base_dir + "/*/")
```

Comment:
So, it would be nice to be able to choose a particular set of directories, but I know you don't run your workflow that way.  I guess if we want to access just a subset, we could run the "gs" function above (that's not a super descriptive name, but ok).